packages/react-positioning/README.md

```diff
@@ -0,0 +1,23 @@
+# @fluentui/react-positioning
+
+A react hook wrapper around [Popper.js](https://popper.js.org/) for Fluent UI.
```

Comment:
Should we restore our "unstable components warning"?

Reply:
you're right, I was too fast in deleting things